### PR TITLE
update of dev-server profile

### PR DESCRIPTION
**Pull request by BrianMarre**
some house keeping for compatibility with the gpu profile

### CODE REVIEW

File: etc/picongpu/bash-devServer-hzdr/cpu_picongpu.profile.example [hunk 1]
```diff
@@ -30,10 +30,10 @@ spack load cmake@3.23.1 %gcc@10.2.0
 spack load openpmd-api@0.14.4 %gcc@10.2.0 \
     ^adios2@2.8.0 \
     ^hdf5@1.12.1 \
-    ^openmpi@4.1.3 +atomics \
+    ^openmpi@4.1.3 +atomics ~cuda\
```
Comment (psychocoderHPC) [inline etc/picongpu/bash-devServer-hzdr/cpu_picongpu.profile.example:33]: I think this line is wrong. Why should we use an MPI without CUDA support? IS is very easy making problems.

**Review by psychocoderHPC — CHANGES_REQUESTED** [review]

File: etc/picongpu/bash-devServer-hzdr/cpu_picongpu.profile.example (hunk 1 above)
Comment (BrianMarre) [reply etc/picongpu/bash-devServer-hzdr/cpu_picongpu.profile.example:33]: because this is cpu-only

Comment (BrianMarre) [reply etc/picongpu/bash-devServer-hzdr/cpu_picongpu.profile.example:33]: there is a separate profile for gpu which explicitly uses cuda

Comment (BrianMarre) [reply etc/picongpu/bash-devServer-hzdr/cpu_picongpu.profile.example:33]: https://github.com/ComputationalRadiationPhysics/picongpu/blob/a17e2aa829e3f36c211959bf94f5d7121a3fba65/etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example#L33

Comment (BrianMarre) [reply etc/picongpu/bash-devServer-hzdr/cpu_picongpu.profile.example:33]: also this is intended as a minimal profile, therefore if we do not need it, it is not there

**Review by psychocoderHPC — DISMISSED** [review]

**Comment by BrianMarre** [comment]
@psychocoderHPC Sorry, found 1 last typo

**Review by psychocoderHPC — APPROVED** [review]

**Pull request merged**